nncf/experimental/torch/fx/statistics/aggregator.py

```diff
@@ -65,6 +66,18 @@ def collect_statistics(self, model: NNCFNetwork, graph: NNCFGraph) -> None:
     def _register_statistics(self, outputs: Dict[str, Tensor], statistic_points: StatisticPointsContainer) -> None:
         return
 
+    @staticmethod
+    def _get_statistic_collector_name(tp: PTTargetPoint, module_to_insert: torch.nn.Module) -> str:
+        """
+        Compouses unique statistic collector name according to given target point and module.
+
+        :param tp: Given target point.
+        :param module_to_insert: Given statistic collection module.
+        :return: Unique statistic collector name according to given target point and module.
+        """
+        tp_name = "_".join([tp.target_node_name, str(tp.input_port_id), str(tp.target_type.value)])
+        return f"{tp_name}_{str(id(module_to_insert))}"
```

Comment:
`"_".join([tp.target_node_name, str(tp.input_port_id), str(tp.target_type.value), str(id(module_to_insert))])`
Looks like it can be done in one line

Reply:
@KodiaqQ just asked me to split this line on two :)